src/backend/distributed/sql/udfs/citus_remote_connection_stats/latest.sql

```diff
@@ -0,0 +1,22 @@
+CREATE OR REPLACE FUNCTION pg_catalog.citus_remote_connection_stats(
+	OUT hostname text,
+	OUT port int,
+	OUT database_name text,
+	OUT connection_count_to_node int)
+RETURNS SETOF RECORD
+LANGUAGE C STRICT
+AS 'MODULE_PATHNAME', $$citus_remote_connection_stats$$;
+
+COMMENT ON FUNCTION pg_catalog.citus_remote_connection_stats(
+	OUT hostname text,
+	OUT port int,
+	OUT database_name text,
+	OUT connection_count_to_node int)
+     IS 'returns statistics about remote connections';
+
+REVOKE ALL ON FUNCTION pg_catalog.citus_remote_connection_stats(
+		OUT hostname text,
+		OUT port int,
+		OUT database_name text,
+		OUT connection_count_to_node int)
+FROM PUBLIC;
```

Comment:
Not for this PR, but we should probably introduce a citus_main_user role that we can grant stuff to. In cloud environments the control plane can then grant the citus_main_user role to the main citus user.